packages/cli/junit.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file should probably be Git-ignored. Perhaps Jest config also needs to be updated, because the file should be generated inside a directory like `__tests__/__results__` rather than at the top level.